Internal: Docs - Remove requirement of putting the command in hook `getId` [ED-4135] (#15589)
elementor/elementor: docs/core/common/assets/js/api/core/hooks.md

            }
        }
        ```
  * Hook conventions 

>     - The hook file name should describe what it does and which group he takes part. For example if the hook is part of navigator mechanism and the hook toggle the navigator it could be called:

Reviewer: ```suggestion
    - The hook file name should describe what it does and where it takes place. For example, if the hook is a part of the navigator mechanism and it toggles the navigator, it could be called:
```

    7  	}
    8  
    9  	getId() {

>     10		return '{FILE_NAME_WITHOUT_JS}';

Reviewer 2: Please fix it in all hooks
